tiny_dnn/util/serialization_functions.h

```diff
@@ -752,6 +769,15 @@ struct serialization_buddy {
   }
 
   template <class Archive>
+  static inline void serialize(Archive &ar, tiny_dnn::selu_layer &layer) {
+    layer.serialize_prolog(ar);
+    ar(cereal::make_nvp("in_size", layer.in_shape()[0]),
+       cereal::make_nvp("lambda", layer.lambda_),
+       cereal::make_nvp("alpha",layer.alpha_)
+       );
+}
```

Comment:
This `}` should be indented. Have you used `clang-format` command?
It should automatically indent the code.